src/frontend/product-video/index.tsx

```
@@ -1,5 +1,5 @@
 import { Modal } from '@wordpress/components';
-import { createRoot, RawHTML, render } from "@wordpress/element";
+import { createRoot, render, useState } from "@wordpress/element";
```

ESLint

[error] Replace `"@wordpress/element"` with `'@wordpress/element'`

(prettier/prettier)